css/dropdown.css

```diff
@@ -0,0 +1,89 @@
+
+.dropdown {
+    min-width: 15px;
+    position: relative;
+    margin: 5px;
+    font-family: "JetBrains Mono", monospace;
+}
+
+.dropdown * {
+    box-sizing: border-box;
+}
+
+.select {
+    background: #7b61ff;
+    color: white;
+    display: flex;
+    justify-content: space-between;
+    align-items:center;
+    border: 5px solid #7b61ff;
+    border-radius: 5px;
+    padding: 5px 10px;
+    transition: 0.3s;
+    font-weight: bold;
+    max-width: 150px;
+    margin: 0 auto;
+}
+
+.select-clicked {
+    border: 2px solid #7b61ff;
+}
```

Comment:
set the same value of `border-width: 2px; `because of different values it looks not very good.
and add `cursor: pointer;` to `.select` class
```
.select {
    background: #7b61ff;
    color: white;
    display: flex;
    justify-content: space-between;
    align-items:center;
    border: 2px solid #7b61ff;
    border-radius: 5px;
    padding: 5px 10px;
    transition: 0.3s;
    font-weight: bold;
    max-width: 150px;
    margin: 0 auto;
    cursor: pointer; 
}

.select-clicked {
    border: 2px solid #7b61ff;
}
```


https://github.com/user-attachments/assets/a27cf92a-be71-4476-b8f6-e92f69021d88